job-descriptions/product-manager.md

```diff
@@ -55,11 +55,10 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
 1. You [apply here](https://jobs.lever.co/sourcegraph/254299f5-f91b-43e2-aa1a-3732963dd296/apply).
 1. We set up a 30-minute call to chat with you about Sourcegraph to find out what you are looking for in your next role.
 1. We give you a writing project to see how you communicate and think about product problems.
-1. We schedule a 2 hour interview to go over your experience in depth and follow up on any questions from the writing project.
-1. We schedule 4 hours of remote interviews over video chat across multiple days.
-   - **Product experience:** We dive deep on how you work as a product manager.
-   - **Working with customers:** The sales and customer engineering teams talk with you about how you work with customer to gather feedback.
-   - **Engineering collaboration:** The engineering team talks with you about how you collaborate and work within a team.
+1. **Product experience:** We review your writing project and dive deep on how you work as a product manager.
```

Comment:
How long is the Product experience call? Should it be more than an hour so there's enough time left for the candidate to ask questions at the end and learn about the product team.

Reply:
It's 1 hour, these are the calls you have been on. I think we just need to do a better job managing time, I'll update the interview prompt to show how much time should be spent on each section.